.github/workflows/docker-build-push-google.yaml

```diff
@@ -140,6 +142,7 @@ jobs:
         uses: docker/build-push-action@v4.0.0
         with:
           push: true
+          platforms: ${{ inputs.platforms }}
```

Comment:
as it is not required this will pass "" or null to the value are you sure this is allowed?